core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1177,7 +1177,7 @@ public function purgeSearchIndex(DataContainer $dc)
 
 		System::getContainer()
 			->get(PageUrlListener::class)
-			->purgeSearchIndex($dc)
+			->purgeSearchIndex($dc->id)
```

Comment:
The method expects an ID and gets a `$dc` object. How can this have worked?

Reply:
It never worked, but didn't get noticed?

Reply:
I guess it just hasn't :)

Reply:
```suggestion
			->purgeSearchIndex((int) $dc->id)
```